src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -708,6 +711,19 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
 
             var grpcEndpointUrl = await _dashboardEndpointProvider.GetResourceServiceUriAsync(context.CancellationToken).ConfigureAwait(false);
 
+            if (dashboardAuthenticationOptions.Value.AllowUnsecureTransport.GetValueOrDefault(false))
+            {
+                // Configure for insecure running.
+                context.EnvironmentVariables["DISABLEAUTHPLACEHOLDER"] = true;
+            }
+            else
+            {
+                // Configure for secure running.
+                context.EnvironmentVariables["BROWSERTOKENPLACEHOLDER"] = dashboardTokenProvider.BrowserToken;
+                context.EnvironmentVariables["OTLPTOKENPLACEHOLDER"] = dashboardTokenProvider.OltpToken;
+                context.EnvironmentVariables["RESOURCESERVERTOKENPLACEHOLDER"] = dashboardTokenProvider.ResourceServerToken;
```

Comment:
needs more underscores

Reply:
They are placeholders, and will be changed to the real values before merge. I just need James' change in first.

Reply:
Which PR? https://github.com/dotnet/aspire/pull/3119 is already merged.

Reply:
Yeah ... it wasn't last night :)

Reply:
This code has been removed in the PR for now. It will go in after I backport this change to P5.